Marlin/src/gcode/feature/L6470/M906.cpp

```diff
@@ -172,58 +247,58 @@ void GcodeSuite::M906() {
     report_current = false;
 
     if (planner.has_blocks_queued() || planner.cleaning_buffer_counter) {
-      SERIAL_ECHOLNPGM("!Can't set KVAL_HOLD with steppers moving");
+      SERIAL_ECHOLNPGM("ERROR - can't set KVAL_HOLD while steppers are moving");
```

Comment:
Note that the word "error" may cause some hosts (Octoprint) to disconnect the serial connection. Unless the error is fatal we should not send lines beginning with "error" to hosts.

Reply:
OK with me.